internal/common/config/config.go

```diff
@@ -73,6 +76,27 @@ type DisasterConfig struct {
 	MagnitudeLambda        float64               // Exponential rate param for disaster magnitude
 }
 
+type IIGOConfig struct {
+	// Executive branch
+	GetRuleForSpeakerActionCost        shared.Resources
```

Comment:
these are all variables with verb names that remind me of function names :(

Reply:
Yeah. The verb is the name of the function that uses the config constant. The verb is the action and "actionCost" was to indicate it was the constant.  

If it's not a high priority, we could look at doing the renaming at a later stage if we have time?

Reply:
sure